packages/vscode-extension/package.nls.json

```diff
@@ -283,8 +283,11 @@
     "teamstoolkit.localDebug.portsAlreadyInUse": "Ports: %s are already in use. Close these ports and try again.",
     "teamstoolkit.localDebug.portWarning": "Changing port(s) in package.json may break local debug. Ensure all port changes are expected or refer to documentation by clicking `Learn More` button. (%s package.json location: %s)",
     "teamstoolkit.localDebug.prerequisitesCheckFailure": "Prerequisites Check Failed. Please check the %s to see the details. If you wish to bypass checking and installing any prerequisites, you can disable them in Visual Studio Code settings.",
-    "teamstoolkit.localDebug.outputPanel": "output panel",
+    "teamstoolkit.localDebug.npmInstallFailure": "NPM Package Install Task Failed. Please check the %s to see the details.",
+    "teamstoolkit.localDebug.outputPanel": "[output panel](command:fx-extension.showOutputChannel)",
```

Comment:
Please add a comment like `_xxx.comment` to note that command doesn't need localization. Only the output panel is translated.

Reply:
Moved the command outof `package.nls.json`